ports/stm32/boards/PYBD_SF2/mpconfigboard.h

```diff
@@ -174,7 +176,7 @@ extern struct _spi_bdev_t spi_bdev2;
 #define MICROPY_HW_SDCARD_DETECT_PIN        (pyb_pin_SD_SW)
 #define MICROPY_HW_SDCARD_DETECT_PULL       (GPIO_PULLUP)
 #define MICROPY_HW_SDCARD_DETECT_PRESENT    (GPIO_PIN_RESET)
-#define MICROPY_HW_SDCARD_MOUNT_AT_BOOT     (0)
+#define MICROPY_HW_SDCARD_MOUNT_AT_BOOT     (1)
```

Comment:
Was it intentional to change these in this PR?

Reply:
these changes were not merged, I removed them